free-courses-hi.md

```diff
@@ -16,11 +16,13 @@
 
 ### C
 
+* [All C Concepts with Free Handwritten Notes | Hindi] (https://youtu.be/ZSPZob_1TOk) - Harry
```

Comment:
title is `C Language Tutorial For Beginners In Hindi (With Notes)`
We need the title as given by the resource so we can avoid duplicates.

Reply:
also, no shortened links.